pypeit/specobjs.py

```diff
@@ -711,6 +712,8 @@ def write_info(self, outfile, pypeline):
             if specobj.RA is not None:
                 objra.append(specobj.RA)
                 objdec.append(specobj.DEC)
+            if specobj.FORCE_EXTRACT is not None:
```

Comment:
also should be done for manual extract objects.

Reply:
Done